api/src/main/kotlin/team/backend/domain/entity/Quiz.kt

```diff
@@ -0,0 +1,45 @@
+package team.backend.domain.entity
+
+import team.backend.infrastructure.BaseEntity
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.EnumType
+import jakarta.persistence.Enumerated
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+
+@Entity
+@Table(name = "tbl_quiz")
+class Quiz(
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "quiz_id")
+    val id: Long? = null,
+
+    @Column(name = "question", nullable = false, columnDefinition="TEXT")
+    val question: String = "",
+
+    @Column(name = "active", nullable = false)
+    val active: Boolean = true,
```

Comment:
What is an `active` column?
Is there no possibility of change?
In my opinion, it seems like it should be a var instead of a val

Reply:
Yes, that's right. Can change it. It's a necessary column for the Quiz table, but I won't change it right away at this stage, so I think it's okay to modify it when ADMIN is added.

Reply:
In addition, from an OOP perspective, it is better to increase the representation by calling a function such as `(un)deActive()` rather than by the name `set()`.